File: app/src/main/java/com/example/ikastyle/Dao/ClothingGearDao.java
package com.example.ikastyle.Dao;

import androidx.room.Dao;
import androidx.room.Query;

import com.example.ikastyle.Entity.ClothingGear;
import com.example.ikastyle.Entity.HeadGear;

import java.util.List;

@Dao
public interface ClothingGearDao {
    @Query("SELECT * FROM MAST_CLOTHING_GEAR WHERE language_code = :languageCode ORDER BY name")
    public List<ClothingGear> getClothingGearList(int languageCode);
}

File: app/src/main/java/com/splatool/ikastyle/model/data/dao/ClothingGearDao.kt
package com.splatool.ikastyle.model.data.dao

import com.splatool.ikastyle.model.data.entity.ClothingGear
import androidx.room.*

@Dao
interface ClothingGearDao {
    @Query("SELECT * FROM MAST_CLOTHING_GEAR WHERE language_code = :languageCode ORDER BY name")
    suspend fun getClothingGearList(languageCode: Int): List<ClothingGear>
}